test/modules/generic-mqtt-tester/src/message_channel.rs

```diff
@@ -118,17 +115,12 @@ impl MessageHandler for RelayingMessageHandler {
         let sequence_number = parse_sequence_number(&received_publication);
 
         info!(
-            "relaying publication with sequence number {}",
-            sequence_number,
+            "relaying publication on topic {} with sequence number {}",
+            received_publication.topic_name, sequence_number,
         );
 
-        // Wait 1 second before relaying to mimic real use cases. This can help
-        // to surface some issues, as relevant connections (i.e. L3 Bridge to L4 Broker)
-        // could drop between original message receipt and subsequent relay.
-        time::delay_for(self.message_frequency).await;
```

Comment:
I removed this because it can cause a potential problem in connectivity tests. Connectivity tests have an `offline period -> online period -> offline period -> online period ...`.

If we use this wait, all messages might not make it back in time to be validated. Since longhaul has a different setup (a message initiator on L4 node), I think that this scenario is covered elsewhere in our tests anyway.